src/buildzqfield.js

```diff
@@ -65,7 +65,7 @@ function is_adx_supported() {
 
 async function buildField(q, name, no_adx, hpp_out, element_hpp_out) {
 
-    if (!no_adx) {
+    if (no_adx == null) {
```

Comment:
I think you have to treat `no_adx` as a boolean.

```suggestion
    if (!no_adx) {
```

Reply:
same argument as above , Fale (could be a valid flag) Null defines no flag at all